pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -352,7 +352,10 @@ def save_checkpoint(self, trainer: "pl.Trainer") -> None:  # pragma: no-cover
         This method runs on all ranks. It is the responsibility of `trainer.save_checkpoint` to correctly handle the
         behaviour in distributed training, i.e., saving only on rank 0 for data parallel use cases.
         """
-        # TODO: unused method. deprecate it
+        rank_zero_deprecation(
+            f"`{self.__class__.__name__}.save_checkpoint()` was deprecated in v1.6 and will be removed in v1.8."
+            " You can use `trainer.save_checkpoint()` instead to manually save a checkpoint."
```

Comment:
```suggestion
            " You can use `trainer.save_checkpoint()` instead of manually save a checkpoint."
```

Reply:
@kaushikb11 your change has a different meaning

Here I'm recommending that the user does `trainer.save_checkpoint()` instead of `mc.save_checkpoint` to manually save a checkpoint